docs/manuals/run-deploy/deploy/azure/index.md

```diff
@@ -1,108 +1,7 @@
-[Azure](https://azure.microsoft.com/en-us), one of the world's leading providers of cloud services,
-is a very relevant choice for deploying Taipy applications. It allows developers to stay focused on
-their application bringing effortless scalability, monitoring and security natively. The following
-documentation shows how to deploy on Azure App Service and get an application into the hands of the
-end-user.
+Taipy applications are designed to be easily deployable and customizable. Once deployed, they can natively support multiple users with authentication and authorization. Taipy is cloud-agnostic and can be deployed on any cloud provider or on-premises.
```

Comment:
This sentence does not bring any value for a reader that is searching how to deploy on Azure.

Reply:
What should I write then? Only the disclaimer like so?
![image](https://github.com/user-attachments/assets/b05b3653-2f06-4c54-b910-573c28dfea30)

Reply:
Something like the following:

``` suggestion
The documentation page is available on demand. Please contact us to get it.
[Contact us](https://taipy.io/book-a-call){: .tp-btn .tp-btn--accent target='blank' }

!!! "Taipy Enterprise Edition"

    Besides this documentation page, Enterprise Edition customers benefit from 
    free-of-charge consulting and support to deploy user applications. Check out the
    [Taipy Enterprise Edition](http://toto.tata.com).
```

Don't forget the capital E to Edition.

Reply:
Understood! I'll do that